rust/feature-flags/src/flags/flag_matching.rs

```diff
@@ -639,12 +639,22 @@ impl FeatureFlagMatcher {
         group_property_overrides: &Option<HashMap<String, HashMap<String, Value>>>,
         hash_key_overrides: Option<HashMap<String, String>>,
     ) -> Result<Option<FeatureFlagMatch>, FlagError> {
-        let flag_property_filters: Vec<PropertyFilter> = flag
+        // Collect ALL property filters - both from regular conditions and super conditions
+        let mut flag_property_filters: Vec<PropertyFilter> = flag
             .get_conditions()
             .iter()
             .flat_map(|c| c.properties.clone().unwrap_or_default())
             .collect();
 
+        // Add super condition properties
+        if let Some(super_groups) = &flag.filters.super_groups {
+            flag_property_filters.extend(
+                super_groups
+                    .iter()
+                    .flat_map(|c| c.properties.clone().unwrap_or_default()),
+            );
+        }
```

Comment:
super conditions are just flag match conditions, so we can extend this filter with no issue.  The problem was that we were only checking flag conditions for overrides, not flag conditions + super conditions.

Reply:
And I assume ordering here doesn't matter? For example, the fact that super filters come before or after the flag property filters is not important?

Reply:
correct; `get_person_overrides` just collects all filters associated with the flag, diffs them against the properties from the request, and then either says "yup I can evaluate this" or "no, I need more info, so I am `None`"